apps/web/src/app/signup/page.tsx

```diff
@@ -99,6 +100,7 @@ export default function SignUp() {
   return (
     <Wrapper>
       <CenteredColumnBox>
+        <LogoComponent />
```

Comment:
Hey, I suggest renaming LogoComponent to Logo. It’s more concise and easier to understand while maintaining consistency with other component names.

Reply:
Done!